package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "@types/enzyme": "3.10.3",
     "@types/jest": "24.0.17",
+    "@types/moment-timezone": "^0.5.12",
```

Comment:
Use an exact version here please.

Reply:
Hi @kgeorgiou, let us know if you have any questions? Would be happy to help! Thanks for your contribution :)

Reply:
Oops sorry, I missed this comment! Will take care of it later today :)